examples/mnist.json

```diff
@@ -1,7 +1,7 @@
 {
   "service": {
     "name": "mnist",
-    "job_url": "https://github.com/springle/tf_examples/archive/master.zip",
+    "job_url": "https://s3.amazonaws.com/downloads.mesosphere.io/tensorflow-dcos/examples/tf_examples-master.zip",
```

Comment:
It looks as if `https://downloads.mesosphere.com/tensorflow-dcos/examples/tf_examples-master.zip` is sufficient here.